blockchain/known_networks.go

```diff
@@ -0,0 +1,71 @@
+package blockchain
+
+import (
+	"github.com/rs/zerolog/log"
+)
+
+// knownNetworks lets us toggle which blockchain client to utilize depending on the supplied Chain ID for the network.
+// Use resources like https://chainlist.org/ or official docs on each chain to distinguish each one.
+var knownNetworks = map[int64]string{
```

Comment:
I'm not in love with this implementation, but it will serve as an easy win for now.